ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -57,3 +57,69 @@ if have_tensorflow_lite
     install_dir: nnstreamer_libdir
   )
 endif
+
+filter_sub_python_sources = [
+  'tensor_filter_python.c',
+  'tensor_filter_python_core.cc'
+]
+
+nnstreamer_filter_python_sources = []
+foreach s : filter_sub_python_sources
+  nnstreamer_filter_python_sources += join_paths(meson.current_source_dir(), s)
+endforeach
+
+if have_python
+  nnstreamer_filter_python_deps = [python_dep, libdl_dep, glib_dep, gst_dep, nnstreamer_dep]
+
+  shared_library('nnstreamer_filter_python2',
+    nnstreamer_filter_python_sources,
+    cpp_args: ['-Wno-sign-compare'],
+    dependencies: nnstreamer_filter_python_deps,
+    install: true,
+    install_dir: filter_subplugin_install_dir
+  )
+
+  static_library('nnstreamer_filter_python2',
+    nnstreamer_filter_python_sources,
+    cpp_args: ['-Wno-sign-compare'],
+    dependencies: nnstreamer_filter_python_deps,
+    install: true,
+    install_dir: nnstreamer_libdir
+  )
+
+  shared_library('nnstreamer_python2',
+    'tensor_filter_python_api.c',
+    name_prefix: '',
+    dependencies: nnstreamer_filter_python_deps,
+    install: true,
+    install_dir: filter_subplugin_install_dir
```

Comment:
Please make sure that installing libnnstreamer_python2.so at this path is Ok. You will need to test it out of gbs/obs. (clean .deb install to your own PC, test out of git repo or build directory)

Reply:
Okay, I've tested the generated libs are successfully moved to the target directory.